projects/cashmere/src/lib/radio-button/radio-button.component.ts

```diff
@@ -139,6 +156,7 @@ export class RadioButtonComponent implements OnInit {
         this.change.emit(new RadioButtonChangeEvent(this, this.value));
     }
 
+    /** Utility function to mark the radio button as changed */
```

Comment:
For clarity, I'd mention the angular change detector here. ...Honestly though this is probably another candidate for an underscore, since its just here so the radio-group.directive can call it.